tests/bug-1142.sil

```diff
@@ -0,0 +1,4 @@
+\begin[class=book,papersize=129mm x 198mm]{document}
```

Comment:
Unless testing the class, papersize, or other spacing things directly (which I don't think is happening in this case) lets stick to `plain` instead of `book` and use `papersize=a6`. Suppressing folios and indentation also helps keep the tests from updating for unrelated changes.

```suggestion
\begin[papersize=a6]{document}
\noindent
\nofolios
```